views/check-your-answers.html

```diff
@@ -9,606 +9,202 @@
-                    </a>
-                </dd>
-            </div>
-            <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
-                    Role and responsibilities
-                </dt>
-                <dd class="govuk-summary-list__value">
-                    <p>Strategic management of the company and the issuing of necessary directives</p>
-                </dd>
-                <dd class="govuk-summary-list__actions">
-                    <a href="/register/managing-officer">
-                        Change
-                        <span class="govuk-visually-hidden"> role and responsibilities</span>
-                    </a>
-                </dd>
-            </div>
-        </dl>
+      {% set formattedPrincipalAddressHtml %}
+        {% if appData.entity.principalAddress.principalAddressLine1 %}
+          {{ appData.entity.principalAddress.principalAddressLine1 }}
+        {% endif %}
+        {% if appData.entity.principalAddress.principalAddressLine2 %}
+          <br>{{ appData.entity.principalAddress.principalAddressLine2 }}
+        {% endif %}
+        {% if appData.entity.principalAddress.principalAddressTown %}
+          <br>{{ appData.entity.principalAddress.principalAddressTown }}
+        {% endif %}
+        {% if appData.entity.principalAddress.principalAddressCounty %}
+          <br>{{ appData.entity.principalAddress.principalAddressCounty }}
+        {% endif %}
+        {% if appData.entity.principalAddress.principalAddressPostcode %}
+          <br>{{ appData.entity.principalAddress.principalAddressPostcode }}
+        {% endif %}
+      {% endset %}
+
+      {% set formattedServiceAddressHtml %}
+        {% if appData.entity.isAddressSameAsPrincipalAddress == 'Yes' %}
+          The correspondence address is the same as the principal or registered office address
+        {% else %}
+          {% if appData.entity.serviceAddress.serviceAddressLine1 %}
+            {{ appData.entity.serviceAddress.serviceAddressLine1 }}
+          {% endif %}
+          {% if appData.entity.serviceAddress.serviceAddressLine2 %}
+            <br>{{ appData.entity.serviceAddress.serviceAddressLine2 }}
+          {% endif %}
+          {% if appData.entity.serviceAddress.principalAddressTown %}
+            <br>{{ appData.entity.serviceAddress.principalAddressTown }}
+          {% endif %}
+          {% if appData.entity.serviceAddress.serviceAddressCounty %}
+            <br>{{ appData.entity.serviceAddress.serviceAddressCounty }}
+          {% endif %}
+          {% if appData.entity.serviceAddress.serviceAddressPostcode %}
+            <br>{{ appData.entity.serviceAddress.serviceAddressPostcode }}
+          {% endif %}
+        {% endif %}
+      {% endset %}
+
+      <h1 class="govuk-heading-xl">Check your answers before sending your application</h1>
+
+      <h2 class="govuk-heading-l">Your details</h2>
+
+      {{ govukSummaryList({
```

Comment:
Probably we will add in the future an https://design-system.service.gov.uk/components/accordion/, the amount of data visualised can be unreadable.

Reply:
Agreed 👍